cmd/helm/installer/options.go

```diff
@@ -71,6 +71,11 @@ type Options struct {
 
 	// EnableHostNetwork installs Tiller with net=host.
 	EnableHostNetwork bool
+
+	// MaxHistory sets the maximum number of release versions stored per release.
+	//
+	// Empty value (0) means no limit.
```

Comment:
- [ ] Since an integer can't be empty, you could rephrase that as, "Zero means no limit."  
  And what if it's negative? Is it the case that a nonpositive value means no limit, or do we trap a negative value before writing to this field?  
  (Later: I see that we ignore a negative value in file _tiller.go_.)